tstest/natlab/firewall.go

```diff
@@ -5,26 +5,67 @@
 package natlab
 
 import (
+	"fmt"
 	"sync"
 	"time"
 
 	"inet.af/netaddr"
 )
 
-type session struct {
+// EndpointDependence expresses
+type FirewallSelectivity int
+
+const (
+	// APDF specifies a destination address-and-port dependent
+	// firewall. Outbound traffic to an ip:port authorizes traffic
+	// from that ip:port exactly, and nothing else.
+	APDF FirewallSelectivity = iota
```

Comment:
I'd prefer to be verbose here instead of just `APDF`. How about `AddressPortDependentFirewall` etc?

Reply:
It makes configuring a firewall quite verbose, and I was trying to avoid that. But you're right that future readability matters more. Fixed.